downstream/modules/platform/proc-controller-adding-new-inventory.adoc

```diff
@@ -0,0 +1,85 @@
+Click the image:search.png[Search,15,15] icon to select from available inventories. 
+Empty groups from input inventories are copied into the constructed inventory.
+* Optional:(Only applicable to constructed inventories): *Cached timeout (seconds)*: Set the length of time you want the cache plugin
+data to timeout.
+* Only applicable to constructed inventories: *Verbosity*: Control the level of output that Ansible produces as the playbook executes related to inventory sources associated with constructed inventories. 
+Select the verbosity from Normal to various Verbose or Debug settings. 
+This only appears in the "details" report view. 
+** Verbose logging includes the output of all commands. 
+** Debug logging is exceedingly verbose and includes information on SSH operations that can be useful in certain
+support instances. Most users do not need to see debug mode output.
+* Only applicable to constructed inventories: *Limit*: Restricts the number of returned hosts for the inventory source associated with the constructed inventory. 
+You can paste a group name into the limit field to only include hosts in that group. 
+For more information, ses the *Source vars* setting.
+* Only applicable to standard inventories: *Options*: Check the *Prevent Instance Group Fallback* option to enable only the instance groups listed in the *Instance Groups* field to execute the job. 
+If unchecked, all available instances in the execution pool will be used based on the hierarchy described in
+[Control where a job runs] in the _Automation Controller Administration Guide_. 
+Click the image:question_circle.png[Help,15,15] icon for additional information.
++
+[NOTE]
+====
+Set the `prevent_instance_group_fallback` option for Smart Inventories through the API.
+====
+
+* *Variables* (*Source vars* for constructed inventories):
+
+** *Variables* Variable definitions and values to apply to all hosts in this inventory. 
+Enter variables using either JSON or YAML syntax. 
+Use the radio button to toggle between the two.
+** *Source vars* for constructed inventories creates groups, specifically under the `groups` key of the data. 
+It accepts Jinja2 template syntax, renders it for every host, makes a `true` or `false` evaluation, and includes the host in the group (from the key of the entry) if the result is `true`. 
+This is particularly useful because you can paste that group name into the limit field to only include hosts in that group. 
+See Example 1 in xref:ref-controller-smart-host-filter[Smart host filters].
+. Click btn:[Save].
+
+After saving the new inventory, you can proceed with configuring permissions, groups, hosts, sources, and view completed jobs, if
+applicable to the type of inventory. 
+
+:context: inventory
+include::proc-controller-adding-permissions.adoc[leveloffset=+1]
+include::proc-controller-add-groups.adoc[leveloffset=+1]
+include::proc-controller-add-hosts.adoc[leveloffset=+1]
+include::proc-controller-add-source.adoc[leveloffset=+1]
+
```

Comment:
To fix the duplicate ID error, you need to close out this context definition. I believe the definition continues until a new one is defined or until this one is closed out. You can do this by restoring the parent context, or simply closing this context out. Since you don't seem to be using a "parent" context, just add the following where you want to close this context out:

:!context: inventory